### ISSUE TITLE
Add YUM repository fails

### ISSUE THREAD

**Issue by andmos** [ocha/ansible-role-dotnet-core#2]
I guess something has changed here, the `Add YUM repository` task is failing: 

```
TASK [DotNetCoreSDK-Linux : .NET | Add YUM repository] ***************************************************************** FAILED - RETRYING: .NET | Add YUM repository (3 retries left).
FAILED - RETRYING: .NET | Add YUM repository (3 retries left).
FAILED - RETRYING: .NET | Add YUM repository (2 retries left).
FAILED - RETRYING: .NET | Add YUM repository (2 retries left).
FAILED - RETRYING: .NET | Add YUM repository (1 retries left).
FAILED - RETRYING: .NET | Add YUM repository (1 retries left).
fatal: [10.0.110.111]: FAILED! => {"attempts": 3, "changed": false, "failed": true, "repo": "packages-microsoft-com-prod", "state": "present"}
fatal: [10.0.110.61]: FAILED! => {"attempts": 3, "changed": false, "failed": true, "repo": "packages-microsoft-com-prod", "state": "present"}
```

Running against CentOS 7 machines.

**Comment by ocha** [comment]
Thanks @andmos. Bug was reproduced on CI and it is fixed in release v2.0.1. Please update your role and try again

**Comment by andmos** [comment]
Great 👍
